src/service/processors.py

```diff
@@ -0,0 +1,51 @@
+from pathlib import Path
```

Comment:
I think we could split the filetree to be mapper (service), reducer (service), worker_server (actual server). That way we pretty much have 3 different blocks

Reply:
I agree it makes sense to split the filetree and just provide "relative" paths, we can talk about that more irl.

What change do you propose here?

Reply:
For now let's leave it as it is, looks good enough